services/cms/src/utils/Gutenberg/UseBlocksWithUnsupportedBlocksRemoved.ts

```diff
@@ -0,0 +1,21 @@
+import { BlockInstance } from "@wordpress/blocks"
+
+export function UseBlocksWithUnsupportedBlocksRemoved(
```

Comment:
```suggestion
export function useBlocksWithUnsupportedBlocksRemoved(
```
Also rename the file